autonomy/services/scaffold/service.yaml

```diff
@@ -7,3 +7,5 @@ license: Apache-2.0
 agent: agent
 network: hardhat
 number_of_agents: 4
+deployment: {}
```

Comment:
what's "deployment" for?

Reply:
We introduced this parameter a while ago for configuring ports for the nodes.